composer.json

```
@@ -32,7 +32,7 @@
         }
     ],
     "require": {
```

```suggestion
    "require": {
        "php": "^7.4|^8.0",
```

@tabacitu, PHP 7.3 **Security** support ends in 6 days (06 Dec 2021), I know Laravel 8 still supports it, but maybe we could/should drop it?

It would allow us to use;
- Short Closures `fn($x) => $x * $x`
- Null coalescing assignment `$column['default'] ??= '-';`
- Typed properties `public string $offerNumber = null;`
- and more ...

All those are in `7.4`, and all those would be handy 🙌

---

That would be cool, but... no 😅  I think it's easier for both us and our users to know we support a certain Laravel version, and all the PHP version it supports.

That's something we can do in the next Backpack version, though. We can ONLY support Laravel 9, which I assume will drop PHP 7.3 support. And that's just a few months away. No point in hurrying it.